Philips.CodeAnalysis.Common/AttributeDefinition.cs

```diff
@@ -13,10 +13,4 @@ public AttributeDefinition(string name, string fullName)
 		public string Name { get; }
 		public string FullName { get; }
 	}
```

Comment:
If a side effect of the analyzer is that only 1 class can exist per file, we should perhaps make that clear in the Analyzer description. Alternatively, update the analyzer to only consider the first class in the file. I think the latter is the preference; if we want to enforce 1 class per file, that should probably be a separate analyzer (so that the consumer has discretion/flexibility to enable or disable.)

Reply:
Created an issue for this